requirements.in

```diff
@@ -3,14 +3,14 @@ celery==5.2.2
 celery-redbeat==2.0.0
 boto3==1.16.63
 dj-database-url==0.5.0
-django==2.2.26
-django-anymail[mailgun]==6.1.0
+django
```

Comment:
shouldn't we at least keep it below the major version upgrade e.g. `django>=3.2, <=4.0`

Reply:
@asadiqbal08 What's your opinion on this?